camunda-commons-ui/lib/analytics/modal.html

```diff
@@ -0,0 +1,84 @@
+<!-- # CE - camunda-bpm-webapp/camunda-commons-ui/lib/analytics/modal.html -->
+<div class="analytics-body">
+    <div class="camunda-logo navigation-element" ng-class="{white: page === 1 || page === 5}" ng-bind-html="logo"></div>
+    <span class="glyphicon glyphicon-remove-sign modal-close" ng-class="{white: page === 1 || page === 5}"
+        ng-click="close()"></span>
+
+    <div class="splash" ng-if="page === 1">
+        <div class="title">
+            {{'TELEMETRY_CAMUNDA_NEEDS_YOU' | translate}}
+        </div>
+
+        <button class="btn btn-default begin" ng-click="next()">
+            {{'TELEMETRY_BEGIN' | translate}}
+        </button>
+    </div>
+
+    <div class="content" ng-if="page === 2">
+        <h3>{{'TELEMETRY_INTRODUCTION_HEADER' | translate}}</h3>
+        <p>{{'TELEMETRY_INTRODUCTION_EXPLANATION_1' | translate}}</p>
+        <p>{{'TELEMETRY_INTRODUCTION_EXPLANATION_2' | translate}}</p>
+        <p>{{'TELEMETRY_INTRODUCTION_EXPLANATION_3' | translate}}</p>
+        <ul class="checkmark">
+            <li>{{'TELEMETRY_INTRODUCTION_COMMUNITY' | translate}}</li>
+            <li>{{'TELEMETRY_INTRODUCTION_IMPROVE' | translate}}</li>
+        </ul>
+
+        <button class="btn btn-default next" ng-click="next()">
+            {{'TELEMETRY_NEXT' | translate}}
+        </button>
+    </div>
+    <div class="content" ng-if="page === 3">
+        <h3>{{'TELEMETRY_DETAILS_HEADER' | translate}}</h3>
+        <ul class="triangle collection-details">
+            <li>
+                <b>{{'TELEMETRY_DETAILS_META_DATA' | translate}}</b>
+                <p>{{'TELEMETRY_DETAILS_META_DATA_WHAT' | translate}}</p>
+                <i>{{'TELEMETRY_DETAILS_META_DATA_WHY' | translate}}</i>
+            </li>
+        </ul>
+        <button class="btn btn-default next" ng-click="next()">
+            {{'TELEMETRY_NEXT' | translate}}
+        </button>
+    </div>
+
+    <div class="content" ng-if="page === 4">
+        <h3>{{'TELEMETRY_PREFERENCES_HEADER' | translate}}</h3>
+        <p>{{'TELEMETRY_PREFERENCES_DETAILS' | translate}}</p>
+        <form class="form-inline">
+            <div class="row">
+                <div class="col-xs-1 col-sm-1 text-right form-control-static">
+                    <input type="checkbox" ng-model="enableUsage" class="form-control usage-statistics"
```

Comment:
Use either:
```suggestion
                    <input type="checkbox" ng-model="$parent.enableUsage" class="form-control usage-statistics"
```
Or use `ng-show` instead of `ng-if`.

See: https://github.com/angular/angular.js/issues/6038

Reply:
Went with `ng-show` and used it everywhere for consistency